README.md

```diff
@@ -6,34 +6,31 @@ This is source code for [summer.nixos.org](https://summer.nixos.org) website.
 To contribute please send a Pull Request. A **preview link** is going to be
 generated and added as a comment once build of the website finishes.
 
-To develop on the website **locally** use the following commands.
+To develop on the website **locally** you have multiple possibilities.
 
 ```console
-$ nix build ./#packages.x86_64-linux.nixos-summer-serve -o ./result-serve
-# or
-$ nix-build . -A packages.x86_64-linux.nixos-summer-serve -o ./result-serve
-
-$ ./result-serve/bin/serve
+# If you use direnv
+$ make serve
+# If you don’t use direnv but have flake-enabled Nix
+$ nix develop -c make serve
+# If you don’t have direnv or flake-enabled Nix
+$ nix-shell --run "make serve"
```

Comment:
We could look at a solution that uses `nix run`. Although this would kind of alienate the non-flake uses a bit.